README.md

```diff
@@ -157,6 +157,12 @@ To skip cleaning the project:
 snapshot --noclean
 ```
 
+To only run tests (i.e. don't actually take any screenshots):
```

Comment:
I think it's okay, when the user does screenshots, it will just be created. We can leave the description like this, I just wanted to be sure I understand it.